From: Juergen Beisert <jbe@pengutronix.de>
Date: Fri, 26 Aug 2011 12:19:33 +0200
Subject: [PATCH] Disable the textengines to be less depended on the Qt
 config.

If the textengines are built, it ends with an error message:

make[3]: Entering directory `[...]/platform/build-target/qwt-6.0.1/textengines/mathml'
compiling qwt_mml_document.cpp
qwt_mml_document.cpp:5:20: fatal error: QDomNode: No such file or directory
compilation terminated.
make[3]: *** [obj/qwt_mml_document.o] Error 1

As the PTXdist rule file only installs the library and not the tools, they can
be disabled.

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>
---
 qwt.pro |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/qwt.pro b/qwt.pro
index 4ab4c08..950c182 100644
--- a/qwt.pro
+++ b/qwt.pro
@@ -13,8 +13,7 @@ TEMPLATE = subdirs
 CONFIG   += ordered
 
 SUBDIRS = \
-    src \
-    textengines
+    src
 
 contains(QWT_CONFIG, QwtDesigner ) {
     SUBDIRS += designer 
